api/test/integration/conftest.py

```diff
@@ -200,12 +200,11 @@ def general_procedure(module: str):
     module : str
         Name of the tested module.
     """
-    base_content = os.path.join(env_path, 'configurations', 'base', '*')
-    module_content = os.path.join(env_path, 'configurations', module, '*')
+    base_content = os.path.join(env_path, 'configurations', 'base')
+    module_content = os.path.join(env_path, 'configurations', module)
     tmp_content = os.path.join(env_path, 'configurations', 'tmp')
-    os.makedirs(tmp_content, exist_ok=True)
-    os.popen(f'cp -rf {base_content} {tmp_content}').close()
-    os.popen(f'cp -rf {module_content} {tmp_content}').close()
+    shutil.copytree(base_content, tmp_content, dirs_exist_ok=True)
+    shutil.copytree(module_content, tmp_content, dirs_exist_ok=True)
```

Comment:
Some modules like rootcheck do not have a specific configuration or files, so the source path could not exist. This is actually the reason why the rootcheck tests [failed](https://github.com/wazuh/wazuh/pull/23514/checks?check_run_id=25394764574):
```
conftest.py:392: in api_test
    general_procedure(module)
conftest.py:207: in general_procedure
    shutil.copytree(module_content, tmp_content, dirs_exist_ok=True)
/usr/lib/python3.10/shutil.py:556: in copytree
    with os.scandir(src) as itr:
E   FileNotFoundError: [Errno 2] No such file or directory: '/tmp/Test_integration_endpoints_B4583_test_files/api/test/integration/env/configurations/rootcheck'
```